learning_resources_search/serializers.py

```diff
@@ -116,6 +112,7 @@ class LearningResourcesSearchRequestSerializer(serializers.Serializer):
         required=False,
         child=serializers.ChoiceField(choices=AGGREGATIONS),
     )
+    id = StringArrayField(required=False, child=serializers.CharField())  # noqa: A003
```

Comment:
I'm not sure how concerned we should be about the warning that id is a python builtin in this instance. I can rename the key to 'ids' or 'learning_resource_id' and add some logic to map that to "id" in the search object